setup.py

```diff
@@ -22,6 +22,7 @@
         "pytz>=2016.10",
         "shortuuid>=0.4.3",
         "tabulate>=0.7.7",
+        "pathlib2; python_version < '3.4'",
```

Comment:
Yes, adding pathlib2 for all installations should be 100% fine. If we do maybe we should change:

```
try:
    from pathlib import PurePath
except ImportError:
    from pathlib2 import PurePath
```

to just:

`from pathlib2 import PurePath`

Since it will be always present anyway, and this way you don't have to worry about supporting two configurations.

Pathlib2's author [states that this is a "simplest approach"](https://github.com/mcmtroffaes/pathlib2/issues/8#issuecomment-162890916):

> It's ok for projects on themselves to rely on pathlib2 only on particular Python versions, however each project by itself should make this decision for itself, and depending on what is used from the pathlib API, that decision may vary across projects. The simplest approach however is simply to depend on pathlib2 for everything, for as long as you need, and only then switch to pathlib when you can drop the compatibility requirements (whenever that is).